app-config.yaml

```diff
@@ -306,3 +307,20 @@ enabled:
   azureDevOps: ${AZURE_ENABLED}
   jenkins: ${JENKINS_ENABLED}
   permission: ${PERMISSION_ENABLED}
+
+# Plugin Confluence
+confluence:
+  # Confluence base URL for wiki API
+  # Typically: https://{org-name}.atlassian.net/wiki
+  wikiUrl: https://org-name.atlassian.net/wiki
```

Comment:
Can you change this to an environment variable named `CONFLUENCE_URL`?  So it should look like this:
```
  wikiUrl: ${CONFLUENCE_URL}
```

Reply:
Done